src/Extensions/FileFormFactoryExtension.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
This contains code from asset-admin https://github.com/silverstripe/silverstripe-asset-admin/pull/1512